parlai/agents/transformer/modules.py

```diff
@@ -819,9 +835,13 @@ def forward(self, x, encoder_output, encoder_mask, incr_state=None):
         )
         x = self.dropout(x)  # --dropout
         x = x + residual
-        x = _normalize(x, self.norm1)
+        if self.variant != 'prelayernorm':
```

Comment:
Explicitly enumerate all of the variants, rather than !=. This is about future proofing.